lib/DDG/Spice/HealthScores.pm

```diff
@@ -0,0 +1,18 @@
+package DDG::Spice::HealthScores;
+# ABSTRACT: Returns restaurant health score information from the 
+#           Eaternet/Swiftype API.
+
+use strict;
+use DDG::Spice;
+
+my $ENGINE_KEY = "_bGXYZx_vQgcy2fNdZzc";
+
+triggers startend => 'health score', 'health scores';
+spice to => 'http://api.swiftype.com/api/v1/public/engines/search.json?q=$1&engine_key=' . $ENGINE_KEY . '&callback={{callback}}';
```

Comment:
Thanks! Just what I needed to know. I'll implement that, and then rebase to get the key out of the PR/branch.